.circleci/config.yml

```diff
@@ -1736,6 +1801,13 @@ workflows:
           sapi: cgi-fcgi
           integration_testsuite: "test_web"
           docker_image: "datadog/dd-trace-ci:php-8.0_buster"
+      - integration_tests:
+          requires: [ 'Prepare Code' ]
+          name: "PHP 81 web tests with nginx + FastCGI"
+          resource_class: medium+
+          sapi: cgi-fcgi
+          integration_testsuite: "test_web"
+          docker_image: "datadog/dd-trace-ci:php-8.1_buster"
       - integration_tests:
           requires: [ 'Prepare Code' ]
           name: "PHP 55 custom autoloaded web tests with nginx + PHP-FPM"
```

Comment:
Commenting here but it referes to https://github.com/DataDog/dd-trace-php/pull/1297/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R1832

I am not aware of this `min_install_tests` test. Should we update it to point to 8.1? Probably @SammyK can provide some context!

Reply:
Guessed we should wait with that until PHP 8.1 is released?

Reply:
sounds good, added to https://datadoghq.atlassian.net/browse/APMPHP-632